src/components/Garden/Stake/StakingMovements.tsx

```diff
@@ -75,10 +79,15 @@ function getCollateralAttributes(status, theme) {
     },
   }
 
-  return collateralAttributes[status]
+  return collateralAttributes?.[status]
```

Comment:
I think we dont need that. Its can mislead the read the code, we dont return undefined.
```suggestion
  return collateralAttributes[status]
```

Reply:
Yep. Will change it.

Reply:
Fixed.